readme.usage.md

```diff
@@ -61,6 +82,7 @@ usage: sesam [-h] [-version] [-v] [-vv] [-vvv] [-skip-tls-verification] [-sync-c
              [command]
 
 Commands:
+  connect   Connects to the target node via Oauth or Tripletex authentication and updates secrets and environmental variables.
```

Comment:
I suggest we call this command `authenticate` as it's really about authenticating against some external service and getting the necessary secrets that are then uploaded into the node - it's not about connecting or authenticating with the Sesam node. 

Thus I would change the description as well:
"Authenticates against the external service of the connector and updates secrets and environment variables (only when working on a connector).

Reply:
done